### PR TITLE
style : add rating

### PR DESCRIPTION
**Pull request by aniketsingh13**
chrome-capture-2022-3-7

### CODE REVIEW

File: Components/Rating.html [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+            <li class="pr-l f-m"><a href="https://www.linkedin.com/in/aniket-singh-3332a5198/" class="nav-icon"
+                    target="blank"><i class="fab fa-linkedin"></i></a>
+            </li>
+        </ul>
+    </nav>
+    <main class="flex mt-xl">
+         <!--aside container  -->
+         <aside class="aside__container font-m f-s ">
+            <ul class="">
+                <li class="p-s "><a class="side_logo f-m font-l " href="../Pages/document.html">Getting Started</a></li>
```
Comment (TheAmlanSaikia) [inline Components/Rating.html:31]: You can add a classname or remove it, don't leave the class field empty.

**Review by TheAmlanSaikia — APPROVED** [review]
Good and clean code. Good to merge. Just left a comment.

File: Components/Rating.html [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+    <link rel="stylesheet" href="../Styles/Rating.css" />
+    <link rel="stylesheet" href="../Styles/main.css" />
+    <link rel="icon" href="../Assets/Screenshot 2022-02-12 222538.png" type="image/png" />
+    <script src="https://kit.fontawesome.com/fa50d5d333.js" crossorigin="anonymous"></script>
+</head>
+<body>
+    <nav class="nav-containerOne flex  p-m">
+        <div class="nav-brand font-xl f-l"><a class="nav-icon-left" href="../index.html">PlayX</a></div>
+        <ul class=" flex flex-center ">
+            <li class="pr-l f-m"><a href="https://github.com/aniketsingh13" class="nav-icon" target="blank"><i
```
Comment (arpit01923) [inline Components/Rating.html]: - replace blank with _blank

File: Components/Rating.html [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+                        class="fab fa-github"></i></a></li>
+            <li class="pr-l f-m"><a href="https://twitter.com/ANIKETS42266714" class="nav-icon" target="blank"><i
```
Comment (arpit01923) [inline Components/Rating.html]: - same here

File: Components/Rating.html [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+                        class="fab fa-twitter"></i></a>
+            </li>
+            <li class="pr-l f-m"><a href="https://www.linkedin.com/in/aniket-singh-3332a5198/" class="nav-icon"
+                    target="blank"><i class="fab fa-linkedin"></i></a>
```
Comment (arpit01923) [inline Components/Rating.html]: same

File: Styles/Rating.css [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+.rating-container{
+    border: 1px solid var(--border-color);
+    width: 100%;
+    height: 18.75rem;
+    gap: .5rem;
+    box-shadow: 1px 2px 1px 1px var(--boxShadow-color);
+    border-radius: 4px;
+}
+.star{
+    color: yellow;
```
Comment (arpit01923) [inline Styles/Rating.css]: use variable here

File: Styles/Rating.css [hunk 6]
```diff
@@ -0,0 +1,23 @@
...
+    cursor: pointer;
+}
+.uncheckd-star{
+    cursor: pointer;
+}
+.uncheckd-star:hover{
+    color: yellow;
```
Comment (arpit01923) [inline Styles/Rating.css]: same here

**Review by arpit01923 — APPROVED** [review]
- clean code
- left comment, work on it
- one suggestion write good description

File: Components/Rating.html (hunk 1 above)
Comment (aniketsingh13) [reply Components/Rating.html:31]: will change it

File: Components/Rating.html (hunk 2 above)
Comment (aniketsingh13) [reply Components/Rating.html]: will change it

File: Components/Rating.html (hunk 4 above)
Comment (aniketsingh13) [reply Components/Rating.html]: will change it

File: Styles/Rating.css (hunk 5 above)
Comment (aniketsingh13) [reply Styles/Rating.css]: will change it

File: Styles/Rating.css (hunk 6 above)
Comment (aniketsingh13) [reply Styles/Rating.css]: will change it

**Pull request merged**